zipline/finance/order.py

```diff
@@ -86,9 +87,16 @@ def to_dict(self):
         if self.broker_order_id is None:
             del dct['broker_order_id']
 
+        dct['sid'] = self.asset
```

Comment:
Can we add a comment here explaining that we use `sid` here rather than `'asset'` for backwards compat with downstream consumers of the perf packet stream?